packages/client/types/constEnums.ts

```diff
@@ -266,7 +266,7 @@ export const enum Providers {
   GITHUB_NAME = 'GitHub',
   GITHUB_DESC = 'Use GitHub Issues from within Parabol',
   GITHUB_SCOPE = 'admin:org_hook,read:org,repo,user,write:repo_hook',
-  GITLAB_SCOPE = 'read_api',
+  GITLAB_SCOPE = 'api',
```

Comment:
Ideally, I'd prefer a less invasive [scope](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html), but `write_repository` or `write_registry` won't let us read the projects. Including one of the read scopes like `read_api,write_repository` is rejected for being a malformed URL even though it follows the same comma separated pattern shown in their [docs](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html).

Reply:
Did you try setting the exact same scopes in the GitLab app? There seems to be an issue with requesting less scope than the app requires

Reply:
Yep, I've played around with a bunch of different scopes and adjusted the GitLab app to require the exact same scopes. 

- When using multiple scopes, like `read_api,write_repository`, the auth page in GitLab says: `The requested scope is invalid, unknown, or malformed.'. 
- When I just used `read_api`, I get an error message saying I can't create an issue. 
- When `api` is the `GITLAB_SCOPE` and it's the scope in the GitLab app, it works for me but you [mentioned in Slack](https://parabol.slack.com/archives/C836NA350/p1648479438025419) that you ran into an error. So I've added `api` and `read_api` to the GitLab app which you mentioned now works for you. You can find the app permissions [here](https://gitlab.com/groups/parabol1/-/settings/applications/209182).

Reply:
to get around that error, try using space separated or comma + space separated values. (`read_api write_repository` or `read_api, write_repository`). OAuth2 spec has some discrepancies.

Reply:
Space-separated works but even with `read_api write_repository write_directory`, it's not possible to create an issue as I get a permission error. So, it looks like `api` is actually the right scope for us